app/js/components/cohorts/cohortModal.js

```diff
@@ -68,13 +69,22 @@ class ActionsComponent extends Component {
                             name :'', 
                             description: ''
                     }));
+                    
                     $('#myCohort').modal('hide');          
                 });
         } else {
             this.setState({ error: "all fields are required" });
         }
     }
 
+    onSave(event) {
+        event.preventDefault();
+        const {name, description} = this.state;
+        if(name && description) {
```

Comment:
Does this mean that if either name or description has data, but not both, then the fields should not be reset?